examples/query/react/basic/src/store.ts

```diff
@@ -1,11 +1,28 @@
-import { configureStore } from '@reduxjs/toolkit'
+import {
+  combineReducers,
+  configureStore,
+  getDefaultMiddleware,
+} from '@reduxjs/toolkit'
+import type { PreloadedState } from '@reduxjs/toolkit'
 import { pokemonApi } from './services/pokemon'
 
-export const store = configureStore({
-  reducer: {
-    [pokemonApi.reducerPath]: pokemonApi.reducer,
-  },
-  // adding the api middleware enables caching, invalidation, polling and other features of `rtk-query`
-  middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware().concat(pokemonApi.middleware),
+const rootReducer = combineReducers({
+  [pokemonApi.reducerPath]: pokemonApi.reducer,
 })
+
+export const setupStore = (initialState?: PreloadedState<RootState>) => {

```

Comment:
grammatically speaking, when considering the function name as using verbs, `setUpStore` is more correct over `setupStore` (`set up` vs `setup`), but it's not a massive deal

e.g. 'We set up the store setup by calling `setUpStore`'

There might be some convention where people just use `setup` regardless, `msw` seems to have gone that route

Reply:
```suggestion
export const setupStore = (preloadedState?: PreloadedState<RootState>) => {
```

Might as well keep this consistent with `configureStore` & the types, and call this `preloadedState`

Reply:
Ah, great catch thank you!

As for `setupStore()` vs. `setUpStore()` I can go either way, personally `setupStore()` just **feels** better but I am not known for my grammar. Looks like RTK does have it's [own method](https://redux-toolkit.js.org/rtk-query/api/setupListeners) `setupListeners()` so perhaps just follow the library for now?

Reply:
good point.